fe/fe-core/src/main/java/org/apache/doris/analysis/IndexDef.java

```diff
@@ -48,12 +52,18 @@ public IndexDef(String indexName, boolean ifNotExists, List<String> columns, Ind
         } else {
             this.comment = comment;
         }
+        if (properties == null) {
+            this.properties = new HashMap<>();
+        } else {
+            this.properties = properties;
+        }
     }
 
     public void analyze() throws AnalysisException {
-        if (indexType == IndexDef.IndexType.BITMAP) {
+        if (indexType == IndexDef.IndexType.BITMAP
+                || indexType == IndexDef.IndexType.INVERTED) {
```

Comment:
I think we can also add `BLOOM_FILTER` check this time, although it is not implemented yet.

Reply:
added